gateway-api/functional/src/test/java/com/project/test/functional/ExampleApiITCase.java

```diff
@@ -0,0 +1,25 @@
+package com.project.test.functional;
+
+
+import static io.restassured.RestAssured.given;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class ExampleApiITCase {
+
+  @Test
+  @DisplayName("Expect 200 status code when endpoint registered.")
+  void callUndefinedRoute() {
+    // @formatter:off
+    given().
+        port(8092).
+        when().
+        get("/healthcheck").
```

Comment:
Why do we not check payments APIs here? I think we should not repeat health check validation - it is validated via Docker HEALTHCHECK.